cmd/geth/retesteth.go

```diff
@@ -888,14 +888,22 @@ func retesteth(ctx *cli.Context) error {
 	vhosts := splitAndTrim(ctx.GlobalString(utils.RPCVirtualHostsFlag.Name))
 	cors := splitAndTrim(ctx.GlobalString(utils.RPCCORSDomainFlag.Name))
 
+	srv := rpc.NewServer()
+
+	err := node.RegisterApisFromWhitelist(rpcAPI, []string{"test", "eth", "debug", "web3"}, srv, false)
+	if err != nil {
+		utils.Fatalf("Could not register RPC apis: %w", err)
+	}
+
+	handler := node.NewHTTPHandlerStack(srv, cors, vhosts)
```

Comment:
Please remove blank lines between declarations of `srv`, `err` and `handler`, and add another one after the `handler` declaration. That way creating the handler is its own paragraph.